xcode/Subconscious/Shared/Library/Func.swift

```diff
@@ -19,7 +19,15 @@ struct Func {
     /// This is useful for working around some of Swift's syntactical
     /// shortcomings. In particular, this lets us treat switch as an
     /// expression that returns a value.
-    static func block<T>(_ closure: () -> T) -> T {
+    static func run<T>(_ closure: () -> T) -> T {
         closure()
     }
+    
+    /// Immediately run the passed throwing closure, returning the result.
+    /// Useful for wrapping switch statements so they can be treated as expressions.
+    static func run<T>(
```

Comment:
Added variant for throwing closures. I find myself using this a lot to turn `switch` statements into expressions, since Swift is not expression-oriented like Rust is.